src/test/java/io/github/christophermanahan/carnitas/HTTPResponseBuilderTest.java

```diff
@@ -0,0 +1,41 @@
+package io.github.christophermanahan.carnitas;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HTTPResponseBuilderTest {
+    @Test
+    void buildsGETResponses() {
+        String statusCode = StatusCodes.GET;
+        String version = Constants.VERSION;
+
+        Response response = new HTTPResponseBuilder()
+          .statusCode(statusCode)
+          .version(version)
+          .build();
+
+        String expectedResponse = version + " " + statusCode + Constants.CRLF
+          + Constants.BLANK_LINE;
+        assertEquals(expectedResponse, new String(response.serialize()));
+    }
+
+    @Test
+    void buildsPOSTResponses() {
+        String statusCode = StatusCodes.POST;
+        String version = Constants.VERSION;
+        String body = "name=<something>";
+
+        Response response = new HTTPResponseBuilder()
+          .statusCode(statusCode)
+          .version(version)
+          .body(body)
+          .build();
+
+        String expectedResponse = version + " " + statusCode + Constants.CRLF
+          + Headers.CONTENT_LENGTH + body.length()
+          + Constants.BLANK_LINE
+          + body;
+        assertEquals(expectedResponse, new String(response.serialize()));
```

Comment:
I wonder if there is an easier way to test this instead of going through `serialize`

Reply:
@damonkelley I haven't figured that out yet, it pops up in other tests as well.